scripts/deps/docker-compose.yml

```diff
@@ -19,7 +19,7 @@ x-postgres-common: &postgres-common
 services:
   zookeeper:
     <<: *container-common
-    image: confluentinc/cp-zookeeper:5.4.3
+    image: confluentinc/cp-zookeeper:7.2.5.arm64
```

Comment:
is this change intended? you are making docker compose arm64 only compatible. IMO you could create a arm version of this docker compose with a name `docker-compose.arm.yml`

Reply:
Sorry, i mistakenly commit this from my mac, i will revert it back to original intel chip version.